### DISCUSSION TITLE
Vitocal 250 DID für die Stellung der internen Pumpe

### DISCUSSION THREAD

**Discussion by cvbasta75** [open3e/open3e#126] [Q&A]
Mahlzeit, hat zufällig jemand die DID für die Stellung der internen Pumpe also ob jetzt Warmwasserspeicher oder Pufferspeicher für den Heizkreis gewärmt wird?

**Comment by cvbasta75** [comment]
Hab mal die DID 2735 gefunden. 0 müsste Ruhestellung sein. 1 Heizkreis und 2 Warmwasser. Vielleicht kann das ja einer bestätigen.

**Comment by MyHomeMyData** [comment]
Ja, DID 2735 benutzte ich auch. Die Zuordnung ist bei meiner Anlage (252-A) aber:
0 => Heizen
1 => Aus
2 => Warmwasserbereitung

Das verwende ich auch in [dieser ](https://github.com/MyHomeMyData/iob.vis.vitocal250) Visualisierung.

**Comment by Philip-Wiege** [comment]
Hi zusammen,

ich hab bei meiner 250-A rausgeforscht:

0: Heizkreis (oder Külen)
1: Interner Puffer (Standby / Abtauung)
2: Warmwasser
3: 50/50 Interner Puffer / Heizkreis
4: 50/50 Interner Puffer / Warmwasser

**Reply by wangnick** [reply]
Die Werte 3 und 4 hab ich auch schon kurzzeitig mal an Ende einer Heizphase oder WW-Phase gesehen.

LG, Sebastian

**Reply by Philip-Wiege** [reply]
Ja genau. In der "Mitte" steht das Ventil auf dem internen Puffer "Links" ist der Heizkreis und "Rechts" Warmwasser. Beim Verstellen wird also automatisch die gemischte Stellung durchfahren.

**Comment by bu-na** [comment]
Alternativ könnte man die aktuelle Stellung des 4/3-Wege-Ventils auswerten (DID 1842):

**Durchströmte Anlagenkomponenten:**
0 % bis < 5 % Sekundärkreis (Heiz-/Kühlkreis 1 oder externer Pufferspeicher)
5 % bis ≤ 25 % Mischbetrieb: Sekundärkreis (Heiz-/Kühlkreis 1 oder externer Pufferspeicher) **und** Integrierter Pufferspeicher
25 % bis 50 % Mischbetrieb: Sekundärkreis (Heiz-/Kühlkreis 1 oder externer Pufferspeicher) **oder** Abtauen
50 % Integrierter Pufferspeicher 
95 % bis 100 % Trinkwassererwärmung

Hier mal ein Hydraulikschema einer Vitocal mit Bivalenzmischer und 4/3-Wegeventil:
![image](https://github.com/user-attachments/assets/e5196236-be96-46e6-aec5-6a1ac9936ff0)
